docs/source/troubleshooting.rst

```diff
@@ -21,7 +21,7 @@ to help!
    `on PyPi <https://pypi.org/project/pyquil/>`_.
 
 2. If the error appears to be authentication-related, refer to the `QCS CLI documentation
-<https://docs.rigetti.com/en/command-line-interface/command-line-interface>`_.
+<https://docs.rigetti.com>`_.
```

Comment:
Keeping this link generic, for the time being. When new docs roll out, we can always make this more specific if we want.

Reply:
Just to be clear -- there are new QCS CLI docs coming?

Reply:
Yep!